# Keybinds
button_enable: 4
axis_linear: 1
axis_angular: 0
button_enable_tank_mode: 2
button_crawler: 1
button_turbo: 5

# Config for rover_teleop:
speed_factor_crawler: 0.01
speed_factor_normal: 0.25
speed_factor_turbo: 1.0 
smallest_radius: 0.30 
